run.go

```diff
@@ -33,9 +33,13 @@ func StartWithSize(c *exec.Cmd, sz *Winsize) (pty *os.File, err error) {
 			return nil, err
 		}
 	}
-	c.Stdout = tty
+	if c.Stdout == nil {
+		c.Stdout = tty
+	}
+	if c.Stderr == nil {
+		c.Stderr = tty
+	}
 	c.Stdin = tty
```

Comment:
Might as well do it for stdin, no?

Reply:
Fixed. Created a new PR: https://github.com/kr/pty/pull/71